gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -62,10 +62,43 @@ <h1 class="title">{{ vulnerability.id }}</h1>
           <dd>{{ vulnerability.published }}</dd>
           <dt>Modified</dt>
           <dd>{{ vulnerability.modified }}</dd>
+          {%- if vulnerability.severity -%}
+          <dt>Severity</dt>
+          <dd>
+            <ul class="aliases">
+              {% for item in vulnerability.severity -%}
+              <li>{{ item.type }} - {{ item.score }}</li>
+              {% endfor -%}
+            </ul>
+          </dd>
+          {%- endif -%}
           <dt>Details</dt>
           <dd class="details">
             {{ vulnerability.details|markdown|safe -}}
           </dd>
+          {% if vulnerability.credits -%}
```

Comment:
Can we move this to after "References" ?

Reply:
updated [694d854](https://github.com/google/osv.dev/pull/2132/commits/694d8547edd24ad486061fa474482b2af3d7ff0b)

you might verify this change via `MAL-2023-8360`.

sample output:
<img width="1045" alt="Screenshot 2024-05-14 at 2 12 05 PM" src="https://github.com/google/osv.dev/assets/73332835/bbd5340b-8eab-4869-a81c-ed7b79ab52fe">